packages/utils/ast-utils.test.js

```diff
@@ -213,14 +213,16 @@ const a = { plugs: [] }
 
 	describe("safeTraverse", () => {
 		it("should safe traverse", () => {
-			const VALUE = "a value";
+			const type = {
+				type: "NodeType"
+			};
 			const p = {
 				value: {
-					value: VALUE
+					value: type
```

Comment:
change the props to `foo`, `bar`, we're not testing props, we're testing the AST values. This would confuse people looking at the tests